app/backend/templates/account_deletion_report.md

```diff
@@ -0,0 +1,22 @@
+---
+subject: "Account deletion reason #{{ reason.id|couchers_escape }}"
+---
+
+{% from "macros.html" import button, link, support_email, email_link, newline %}
+
+Someone deleted their account and wrote a reason.
+
+
+* Reason{{ newline(html)|couchers_safe }}
```

Comment:
Post-merge comment: Remove all `{{ newline(html)|couchers_safe }}`, they make the template unreadable. Reports should do fine with markdown:s html generator.

Reply:
Oh... then we can switch to an easier syntax ;)

Reply:
Yes, that'd be good! I would be keen to change it; the current templates are a bit of a pain. But it works for now... We can also have a look when we redo the emails soon-ish when the notifications start being more complete.